public/content/en/basics/structs.md

```diff
@@ -28,7 +29,8 @@ is defined through a `this(...)` member function:
         }
             ...
 
-    Person p(30, 180);
+    Person p(30, 180); // initialization
+    t = Person(30, 180);  // creation
```

Comment:
Ah now I get it. We could make it clearer by chsnging the comment to "assignment to new instance"?